satellite/rewards/money.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package rewards
+
+import (
+	"fmt"
+)
+
+// USD describes USD currency in cents.
+type USD struct {
```

Comment:
Maybe it's just me but `USD` has the implication that the unit would be dollars? Should we just use `Cents` ?

Reply:
The comment can be probably improved. Since this is an opaque type (it doesn't show how it stores things internally), it really doesn't matter how it stores things... at least for code outside.